ui/components/MesheryPerformance/Dashboard.js

```diff
@@ -113,7 +113,7 @@ function Dashboard({
                   </div>
                 </div>
               </Grid>
-              <div style={{ height : "10.4rem", width : "1px", background : "black", top : "11.2rem", bottom : "0", position : "absolute", left : "36%", backgroundColor : "#36454f", opacity : "0.7" }}></div>
+              <div style={{ height : "10.4rem", width : "1px", background : "black", marginTop : "1.1rem", bottom : "0" , left : "36%", backgroundColor : "#36454f", opacity : "0.7" }}></div>

```

Comment:
Is there a central theme that can be referenced, @Tanuj22, for these colors?

If so, will you kindly update the contributing-ui docs with a link to it?